Sources/Multitool/Options.swift

```diff
@@ -26,7 +26,8 @@ public class Options {
             set { _build = newValue }
         }
 
-        private var _build = getenv("SWIFT_BUILD_PATH")
+        private var _Packages: String?
```

Comment:
Aha, ok then :) 
I also noticed that `_Packages` is never used. Is is suppose to be so or should it be use in 
`public var Packages: String` ?